fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -51,12 +51,15 @@ def simulators(requested_os_type = "")
       def connected_devices(requested_os_type)
         UI.verbose("Fetching available connected devices")
 
+        tvos_devices = ["AppleTV"]
+        ios_devices = ["iPhone", "iPad", "iPod"]
+
         device_types = if requested_os_type == "tvOS"
-                         ["AppleTV"]
+                         tvos_devices
                        elsif requested_os_type == "iOS"
-                         ["iPhone", "iPad", "iPod"]
+                         ios_devices
                        else
-                         []
+                         ios_devices + tvos_devices
```

Comment:
Is this behavior change intentional? i.e., changing from returning an empty array in the case of non matching os_type to returning all device types?